lib/ensemble_theme.dart

```diff
@@ -234,7 +234,8 @@ class EnsembleTheme {
   static const Color nearlyBlue = Color(0xFF00B6F0);
   static const Color nearlyBlack = Color(0xFF213333);
   static const Color grey = Color(0xFF3A5160);
-  static const Color dark_grey = Color(0xFF313A44);
+  static const Color darkGrey = Color(0xFF313A44);
+  static const Color primary = Color(0xFF08B48F);
```

Comment:
Remove this. don't define any color scheme. We have the global scheme that will change the entire app's look and feel. All widgets should follow this.